### PR TITLE
Update sensitivity-labels.md

### PR DESCRIPTION
**Pull request by anandab-msft**
updated docs to reflect users can use beta API to update sensitivity label.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/sensitivity-labels.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ Before you use sensitivity labels for Teams, be aware of the following limitatio
 - **Sensitivity labels aren't supported by Teams Graph APIs, PowerShell cmdlets, and templates**
     
     Users won't be able to apply sensitivity labels on teams that are created directly through Teams Graph APIs, Teams PowerShell cmdlets, and Teams templates.
+    
+    However, users can use [update groups beta API](https://docs.microsoft.com/en-us/graph/api/group-update?view=graph-rest-beta&tabs=http) to update sensitivity label.
```
Comment (JohanFreelancer9) [inline Teams/sensitivity-labels.md:88]:
```suggestion
    However, users can use the [update groups beta API](https://docs.microsoft.com/graph/api/group-update?view=graph-rest-beta&tabs=http) to update sensitivity labels.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

File: Teams/sensitivity-labels.md (hunk 1 above)
Comment (anandab-msft) [reply Teams/sensitivity-labels.md]: @JohanFreelancer9 - I clicked on commit  suggestions, i am assuming that will automatically commit the changes suggested by you.

**Review by JohanFreelancer9 — APPROVED** [review]

File: Teams/sensitivity-labels.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Teams/sensitivity-labels.md]: Yes, that is perfect. Thank you so much.

**Comment by scanum** [comment]
@robmazz PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by robmazz** [comment]
@scanum  and @JohanFreelancer9 - this new link (https://docs.microsoft.com/en-us/graph/api/group-update?view=graph-rest-beta&tabs=http) should be a relative link to support private cloud tenants. Please update to the following (/graph/api/group-update?view=graph-rest-beta&tabs=http) and I'll merge, thanks!

**Comment by anandab-msft** [comment]
> @scanum and @JohanFreelancer9 - this new link (https://docs.microsoft.com/en-us/graph/api/group-update?view=graph-rest-beta&tabs=http) should be a relative link to support private cloud tenants. Please update to the following (/graph/api/group-update?view=graph-rest-beta&tabs=http) and I'll merge, thanks!

@scanum - can you pls update the link as asked by @robmazz

**Comment by scanum** [comment]
@Reeced40 Could you please help me update the link? Thanks!

File: Teams/information-barriers-in-teams.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Information barriers (IBs) are policies that an admin can configure to prevent i
 
 >[!NOTE]
 >- Information barrier (IB) groups cannot be created across tenants.
->- Using bots, Azure Active Directory (Azure AD) apps, and some APIs to add users is not supported in version 1.
+>- Using bots, Azure Active Directory (Azure AD) apps, API to send activity feed notifications and some APIs to add users is not supported in version 1.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:27]:
```suggestion
>- Using bots, Azure Active Directory (Azure AD) apps, API to send activity feed notifications, and some APIs to add users is not supported in version 1.
```

File: Teams/sensitivity-labels.md [hunk 3]
```diff
@@ -84,6 +84,8 @@ Before you use sensitivity labels for Teams, be aware of the following limitatio
 - **Sensitivity labels aren't supported by Teams Graph APIs, PowerShell cmdlets, and templates**
     
     Users won't be able to apply sensitivity labels on teams that are created directly through Teams Graph APIs, Teams PowerShell cmdlets, and Teams templates.
+    
+    However, users can use the [update groups beta API](https://docs.microsoft.com/graph/api/group-update?view=graph-rest-beta&tabs=http) to update sensitivity labels.
```
Comment (JohanFreelancer9) [inline Teams/sensitivity-labels.md:88]:
```suggestion
    However, users can use the [update groups beta API](/graph/api/group-update?view=graph-rest-beta&tabs=http) to update sensitivity labels.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@anandab-msft - could you please commit the minor additional edit and the relative link, as requested. Many thanks.

**Comment by Reeced40** [comment]
@anandab-msft please can you review merge conflict?

**Review by JohanFreelancer9 — APPROVED** [review]
Copy is fine.

**Review by scanum — APPROVED** [review]

**Comment by tiburd** [comment]
Closing this PR because it's old, has file conflicts, and hasn't had any recent activity.

@anandab-msft - if you still want to propose the changes in this PR, please open a new PR in the private repo.

Thanks!

cc: @scanum

**Pull request closed without merging**